src/routes/Weather/modules/weather.js

```diff
@@ -0,0 +1,44 @@
+import axios from 'axios';
```

Comment:
can we rename this file to `weatherDucks.js` and put it in the root of the Weather folder?